subsys/bluetooth/mesh/Kconfig

```diff
@@ -1109,6 +1111,7 @@ endif
 
 menuconfig BT_MESH_DFD_SRV
 	bool "Support for Firmware Distribution Server model"
+	select BT_MESH_MODEL_EXTENSIONS
```

Comment:
Reverse dependency isn't required here, since extension will be added via DFU client dependency.